contracts/mocks/integration/IntegrationMockStakingPool.sol

```diff
@@ -10,47 +10,20 @@ import "../../modules/staking/StakingPool.sol";
 
 contract IntegrationMockStakingPool is StakingPool {
 
-  /* immutables */
-  address public immutable memberRoles;
-
-  mapping (uint => uint) public usedCapacity;
-  mapping (uint => uint) public stakedAmount;
-
-  mapping (uint => uint) public mockPrices;
-
   constructor (
     address _nxm,
     address _coverContract,
-    ITokenController _tokenController,
-    address _memberRoles
+    ITokenController _tokenController
```

Comment:
The constructor doesn't seem to add anything useful, let's remove it entirely

Reply:
the inheritance means you need a constructor so i left it like this